src/database/DataTypes/Options.js

```diff
@@ -4,7 +4,16 @@
  */
 
 import Realm from 'realm';
-
+/**
+ * An Options object is a simple explanation of why a change or
+ * value was used in a situation. Used for stock takes, to
+ * provide a reason for having a stock take batch quantity differ
+ * from the stock take batch snapshot quantity.
+ * @property  {string}  id
+ * @property  {string}  title     A generic name field
+ * @property  {string}  type      The type of option. See below for types
+ * @property  {boolean} isActive  If the option is currently available to use
+ */
 export class Options extends Realm.Object {}
 
 export default Options;
```

Comment:
should the export be after options schema defined below ? I guess this is done in Stocktake PR ?

Reply:
done a1ea4e47e7ac0b926851fe89977a67cda9730d65